packages/gatsby/cache-dir/loader.js

```diff
@@ -270,8 +271,8 @@ export class BaseLoader {
 
       const finalResult = {}
 
-      const componentChunkPromise = this.loadComponent(componentChunkName).then(
-        component => {
+      const createLoadComponentPromise = () =>
```

Comment:
Nit: why not put this arrow in the `else` branch below? Now it will create the arrow and may end up not using it.

Reply:
yeah good point — I was really tired yesterday 😅